api/openapi-spec/swagger.json

```diff
@@ -919,6 +1044,13 @@
           "description": "DAG template subtype which runs a DAG",
           "$ref": "#/definitions/io.argoproj.workflow.v1alpha1.DAGTemplate"
         },
+        "initContainers": {
+          "description": "InitContainers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/io.argoproj.workflow.v1alpha1.InitContainer"
```

Comment:
Why not just 
`"$ref": "#/definitions/io.k8s.api.core.v1.Container"`
?

Reply:
Thank you for the comment.
It's because I wanted to mirror volume mounts as we do for sidecars.
https://github.com/argoproj/argo/pull/1183/files#diff-67c8e7d98efdde2ba49de3637b0f435bR457

Reply:
In that sense, I think we can rename `Sidecar` struct to something like `UserContainer` and use it for both sidecar and init containers.

Reply:
@alexmt I'd also like to have your opinion about this.

Reply:
Note that `SideCar` still references the `v1.Container` as I proposed.

Reply:
> In that sense, I think we can rename `Sidecar` struct to something like `UserContainer` and use it for both sidecar and init containers.

I think this would be the best. Copying the whole Kubernetes definition is not great for maintenance and compatibility.

Reply:
I updated the PR!